### ISSUE TITLE
Remove unsupported/never supported plugins from the repo

### ISSUE THREAD

**Issue by skurvish** [joomlahenk/fabrik#457]
I wonder if we should remove them from the repo so users won't try and install them not think we might work on them.

**Comment by joomlahenk** [comment]
Yes, I think we should remove the never supported ones. Certainly the ones that don't work on J!3.

**Comment by lcollong** [comment]
I'd like to push up the form's log plugin. The new one made by Hugh. Although it has a bug not allowing the others plugin to run after it. Specially if there is an onAfterProcess php plugin "after" (in the form's plugins list) the log plugin. But knowing this I used it a lot on J3/F3 site without problems.
I've never used the ones left blank in the wiki list of plugins. So it'll be ok for me to remove them.

**Comment by trob** [comment]
I'd like to keep them except the ones definitively not working in F3. There may be 'useful' ones we didn't touch until now (e.g. tags element or article form) because WE don't use them but they may be used in other F3 applications.

But
If a user wants to use and test one of the "blank" plugins they should 
first test if it's running on F3/php8
give exact setup of all parameters
give exact error descriptions (or confirm that it's working)

**Comment by skurvish** [comment]
Yes, you are right.

**Comment by skurvish** [comment]
So, this list: 

Element: attending, button, rating spotify, youtube
Form: confirmation
List: download
Visualization: calendar, fusion_gant_chart

**Comment by joomlahenk** [comment]
What about system\fabrikj2store and community\fabrik ?
Do we need search\fabrik ?

**Comment by skurvish** [comment]
Do we know if they do not work in F3?

**Comment by joomlahenk** [comment]
I never tried

**Comment by juuser99** [comment]
I would definitely keep "button" element and search\fabrik plugin.

**Comment by zmtk-josecardoso** [comment]
Hello guys. Community Builder is a must have!

**Comment by skurvish** [comment]
Perhaps you would like to test it and get it working?

**Comment by lousyfool** [comment]
**@zmtk-josecardoso:**

The F3 CB plugin is outdated since long. But, also since long, there is absolutely no need for it anymore.
Fabrik's content plugin works in CB, and vice versa. Fabrik's placeholders work in CB and, vice versa, CB's substitutions work in Fabrik. This is giving you maximum flexibility to let you do what you need without depending on any "special" plugin.

(Note: as usual, you'll need to enable "prepare content" or "process Joomla plugins" or whatever else it might be called in other cases.)

So, no reason at all anymore for any CB-related plugin in Fabrik.
